app/views/articles/show.html.erb

```diff
@@ -1,6 +1,13 @@
 <h1><%= @article.title %></h1>
 <p><%= @article.body %></p>
-<p>Author: <%= @article.user.username %></p>
+<p>
+  Author:
+  <% if @article.user.nil? %>
```

Comment:
Find a way to refactor this

Hint: create helper method.

Reply:
Done.